### DISCUSSION TITLE
How to switch to another language?

### DISCUSSION THREAD

**Discussion by karjalankivi** [netbox-community/netbox#15589] [Help Wanted!] [answered]
Hi! 
I'm trying to switch to any language (user/preferences/language: russian or some other) in the web-interface of my netbox but it doesn't work. I have tried clean data and cookies in the browser (Chrome and Edge) and reboot dockers images and node but it still didn't help. Also cheked configuration.py and settings.py.
Netbox - last update. 

Btw, this applies not only to changing the language. I tried to change to dark mode and it also doesnt apply

**Comment by candlerb** [comment, marked as answer]
Netbox version?

Dark mode works for me in v3.7.4, using the mode selection in the user menu:

![image](https://github.com/netbox-community/netbox/assets/44789/c16498ba-8633-4d50-af7a-320965ea3ae4)

For translations, you need to set [ENABLE_LOCALIZATION](https://docs.netbox.dev/en/stable/configuration/system/#enable_localization) = True in configuration.py, and restart. Then you can change the language using the "Preferences" option in the same menu.

However, I'm using a regular Netbox install, not docker.

The docker configuration only allows certain settings to be picked up from environment variables, and ENABLE_LOCALIZATION is [not one of them](https://github.com/netbox-community/netbox-docker/blob/2.8.0/configuration/configuration.py).  So you will need to create a file extra.py and attach it as a volume, as documented [here](https://github.com/netbox-community/netbox-docker/wiki/Configuration#custom-configuration-files).

**Reply by karjalankivi** [reply]
The version is v3.7.4 also.
You're right. Thank you! 
I needed to add this value **[ENABLE_LOCALIZATION]** specifically to the docker folder (~/netbox-docker/configuration/extra.py) but I imagined it was already enabled by default...
